package proto_ff;

import "yd_fieldoptions.proto";


message headPortraitheadattributeDesc
{
	optional int32 type = 1[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message headPortraithead
{
	optional int64 ID = 1[(yd_fieldoptions.field_cname) = "头像ID"];
	optional string name = 2[(yd_fieldoptions.field_cname) = "头像名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 quality = 3[(yd_fieldoptions.field_cname) = "头像品质"];
	optional string professionLimit = 4[(yd_fieldoptions.field_cname) = "职业限制", (yd_fieldoptions.field_bufsize) = 60];
	optional int64 activationItem = 5[(yd_fieldoptions.field_cname) = "激活道具ID"];
	optional int32 activationNum = 6[(yd_fieldoptions.field_cname) = "激活道具数量"];
	optional int64 starId = 7[(yd_fieldoptions.field_cname) = "升星道具id"];
	optional string starNum = 8[(yd_fieldoptions.field_cname) = "升星道具数量", (yd_fieldoptions.field_bufsize) = 200];
	optional int32 starUp = 9[(yd_fieldoptions.field_cname) = "升星等级上限"];
	optional int32 starBer = 10[(yd_fieldoptions.field_cname) = "每次升星属性增加的百分比值"];
	repeated headPortraitheadattributeDesc attribute = 11[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 6];
}


message Sheet_headPortraithead
{
	repeated headPortraithead headPortraithead_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message headPortraitframeattributeDesc
{
	optional int32 type = 1[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message headPortraitframe
{
	optional int64 ID = 1[(yd_fieldoptions.field_cname) = "边框ID"];
	optional string name = 2[(yd_fieldoptions.field_cname) = "边框名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 quality = 3[(yd_fieldoptions.field_cname) = "边框品质"];
	optional string professionLimit = 4[(yd_fieldoptions.field_cname) = "职业限制", (yd_fieldoptions.field_bufsize) = 60];
	optional int64 activationItem = 5[(yd_fieldoptions.field_cname) = "激活道具ID"];
	optional int32 activationNum = 6[(yd_fieldoptions.field_cname) = "激活道具数量"];
	optional int64 starId = 7[(yd_fieldoptions.field_cname) = "升星道具id"];
	optional string starNum = 8[(yd_fieldoptions.field_cname) = "升星道具数量", (yd_fieldoptions.field_bufsize) = 200];
	optional int32 starUp = 9[(yd_fieldoptions.field_cname) = "升星等级上限"];
	optional int32 starBer = 10[(yd_fieldoptions.field_cname) = "每次升星属性增加的百分比值"];
	repeated headPortraitframeattributeDesc attribute = 11[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 6];
}


message Sheet_headPortraitframe
{
	repeated headPortraitframe headPortraitframe_List = 1[(yd_fieldoptions.field_arysize)=20];
}

message headPortraitbubbleattributeDesc
{
	optional int32 type = 1[(yd_fieldoptions.field_cname) = "类型"];
	optional int32 value = 2[(yd_fieldoptions.field_cname) = "值"];
}

message headPortraitbubble
{
	optional int64 ID = 1[(yd_fieldoptions.field_cname) = "气泡ID"];
	optional string name = 2[(yd_fieldoptions.field_cname) = "气泡名称", (yd_fieldoptions.field_bufsize) = 60];
	optional int32 quality = 3[(yd_fieldoptions.field_cname) = "气泡品质"];
	optional string professionLimit = 4[(yd_fieldoptions.field_cname) = "职业限制", (yd_fieldoptions.field_bufsize) = 60];
	optional int64 activationItem = 5[(yd_fieldoptions.field_cname) = "激活道具ID"];
	optional int32 activationNum = 6[(yd_fieldoptions.field_cname) = "激活道具数量"];
	optional int64 starId = 7[(yd_fieldoptions.field_cname) = "升星道具id"];
	optional string starNum = 8[(yd_fieldoptions.field_cname) = "升星道具数量", (yd_fieldoptions.field_bufsize) = 200];
	optional int32 starUp = 9[(yd_fieldoptions.field_cname) = "升星等级上限"];
	optional int32 starBer = 10[(yd_fieldoptions.field_cname) = "每次升星属性增加的百分比值"];
	repeated headPortraitbubbleattributeDesc attribute = 11[(yd_fieldoptions.field_cname) = "属性", (yd_fieldoptions.field_arysize) = 6];
}


message Sheet_headPortraitbubble
{
	repeated headPortraitbubble headPortraitbubble_List = 1[(yd_fieldoptions.field_arysize)=20];
}
